access-ocp-api-through-cis.md

```diff
@@ -0,0 +1,104 @@
+## Accessing the api endpoint deployed on OpenShift through CIS
+
+1. Setup an OpenShift cluster with ingress. The cluster comes with a default endpoint `/healthz` that is accessible as:
+   ```
+   % oc get routes -n openshift-ingress
+   NAME                                      HOST/PORT                                                                               PATH       SERVICES                  PORT   TERMINATION     WILDCARD
+   route.route.openshift.io/router-default   router-default.xxx-3b5bf5f75xxxx21c8c35ad277-0000.us-south.containers.appdomain.cloud   /healthz   router-internal-default   1936   edge/Redirect   None
+
+   % curl https://router-default.xxx-3b5bf5f75xxxx21c8c35ad277-0000.us-south.containers.appdomain.cloud/healthz
+   ok
+   ```
+
+   Alternatively you deploy your app and create the NodePort service for the application endpoint.
```

Comment:
I think users need context about why you would choose this alternative. 

Does this topic help users understand the different ways? https://cloud.ibm.com/docs/openshift?topic=openshift-cs_network_planning

Reply:
Deploying an app and exposing the app is what customers would do.  But we should mention that only for this doc we are using the existing end point which is the health check endpoint /healthz (similarly they would have an endpoint they deploy an app)

Reply:
Rearranged the text. Hope it should be good now.